src/constants/examples.constant.ts

```diff
@@ -1,6 +1,10 @@
 import { ENUMS } from './enum.constant';
+import { MAX_FILE_SIZE_BYTES } from './max-file-size-bytes.constant';
 
 export const EXAMPLES = {
   SITE_ID: '12345678',
   SITE_STATUS_CODE: ENUMS.SITE_STATUSES.CREATED,
+  FILE_SIZE: MAX_FILE_SIZE_BYTES,
+  FILE_NAME: 'document.txt',
+  FILE_LOCATION_PATH: 'folder/subfolder',
```

Comment:
As the Swagger page auto-fills with these values, it would be nice for testing purposes to use the real file name and path of a file in DFTS dev storage, but does this pose any kind of privacy risk? @abhi-markan

Reply:
Can these be loaded from .env?

Reply:
Great idea, I've made that change